### ISSUE TITLE
Clarify Browser compile-time options

### ISSUE THREAD

**Issue by mgroeber9110** [bluewaysw/pcgeos#648]
One cleanup task I would like to try is to radically simplify the compile-time options (especially `options.goh`) of WebMagick again.

There are lots of build and product variants that we are probably no longer interested in (GPC, NDO, Yago, ...), so I would like to remove most of the conditional processing of options and have one straightforward options file that just enables all the options we want to keep in FreeGEOS, and perhaps documents the ones we have deactivated.

This includes things like:

 - Keep the spinning globe animation?
 - Kids Guard
 - Which icons should be in the toolbar?
 - What settings can be changed by the user?
   - What are the default values of these settings?
 - Links to other apps (e.g. HTML Editor)

I would ideally like to keep most of the features that were added at some point, but perhaps remove a few project-specific variants that no longer have any use (e.g. EarthLink setup, alternative cache for specific hosts, perhaps even the spinning globe) so that we get a better view of what features we want to preserve in WebMagick.

**Comment by HubertHuckevoll** [comment]
Hi Marcus, I've already worked on this quite a bit but it's not in the master yet. I especially cleaned up the goh file with the ui components which was quite a task. I also fixed a tricky bug in the MDI handling of the browser along the way. However, I died exactly at the point where I had to decide, about how to go on with the options.goh.

**Comment by HubertHuckevoll** [comment]
My work is here, just let me know if you want to take over - so we don't work on the same stuff twice...

https://github.com/HubertHuckevoll/pcgeos/tree/browser-ui-multi

**Comment by mgroeber9110** [comment]
If you already started on this, perhaps we can use this ticket to discuss what could be done. My strategy would have been roughly like this:

- Move the few fixed options from local.mk into options and remove the conditional defines in the .gp file
- Decide which PRODUCT to use as the basis (NDO or GPC), probably trying to use the same as BBE
  - This brings up an interesting question: I noticed that we currently compile with PRODUCT_ (i.e. no name). Is this intentional, and does it match the latest BBE, which I guess is what we want to recreate? Or did we somehow lose the correct PRODUCT define in some cleanup operation?
- Move all defines not part of this variant (e.g. "Yago") into comments, so we see what is left.

Then, we can gradually decide if we want to bring some product-specific variants back because they demonstrate interesting ideas (e.g. GPC local pages?).

We also have the GPC browser as a separate app that we may want to remove at some point, because it seems to be mostly a fork of the BBX browser that we no longer maintain.

**Comment by mgroeber9110** [comment]
For reference, here is a screenshot of WebMagick 3.0 menus from BBE 4.13, so we can see what features were included:

![grafik](https://github.com/user-attachments/assets/8e25cfc7-f69d-4e4b-8015-9bea241d37ad)

**Comment by mgroeber9110** [comment]
Obsolete after #762 was merged.
